english/escape-to-the-wilderness/wilderness.md

```diff
@@ -0,0 +1,22 @@
+You decide to escape the city for a weekend and head out to the wilderness.
+
+With your bag all packed with camping gear, you make your way toward the mountains.
+
+The hike to a campsite was relatively easy so you settle down in a sleeping bag 
+next to the campfire, and drift off to sleep. 
+
+You awake to find you're being dragged across the forest floor. But by who or what?
+
+Perhaps a [bear](https://github.com/udacity/create-your-own-adventure/blob/master/english/bear/bear.md) got a hold of you.
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.